### ISSUE TITLE
devtools.switchToFrame errors on null id when there is no parentFrame

### ISSUE THREAD

**Issue by straker** [webdriverio/webdriverio#6607]
**Environment (please complete the following information):**
 - **WebdriverIO version:** 7.2.1
 - **Mode:** Standalone
 - **Node.js version:** 12.18.2
 - **NPM version:** 6.14.5
 - **Browser name and version:** Chrome 89.0.4389.82
 - **Platform name and version:** MacOs Catalina 10.15.7

**Config of WebdriverIO**
N/A

**Describe the bug**
When passing a `null` id to `client.switchToFrame` and the page does not have a parent frame, the code errors

```js
(node:12085) UnhandledPromiseRejectionWarning: Cannot read property 'element-6066-11e4-a52e-4f735466cecf' of null
TypeError: Cannot read property 'element-6066-11e4-a52e-4f735466cecf' of null
    at DevToolsDriver.switchToFrame (/node_modules/devtools/build/commands/switchToFrame.js:31:36)
```

**To Reproduce**
Run the following script

```js
const { remote } = require('webdriverio')
;(async () => {
  const client = await remote({
    services: ['chromedriver'],
    capabilities: {
      browserName: 'chrome'
    }
  })

  await client.url('https://google.com')

  // why id is `null` doesn't matter, let's just say it's returned from a function call that does 
  // some work and returns frame ids
  await client.switchToFrame(null);
  await client.deleteSession()
})()
```

**Expected behavior**
`switchToFrame` should allow `null` ids and not error

**Additional context**
`switchToFrame` only handles `null` ids when there is a `parentFrame` on the page, otherwise it continues on as though the `id` was not null. For refrence, `switchToParentFrame` noops when there is no `parentFrame`.

**Comment by erwinheitzman** [comment]
It does allow `null` because null represents the top-level browsing context (i.e., not an iframe). So when there's no top level you'll receive an error which I'd say is to be expected.

**Comment by erwinheitzman** [comment]
The following example further demonstrates this:
```js
const { remote } = require('webdriverio');

let browser;

(async () => {
	browser = await remote({
		automationProtocol: 'devtools',
		capabilities: {
			browserName: 'chrome',
			logLevel: 'debug',
			'wdio:devtoolsOptions': {
				headless: false,
			}
		}
	})

	await browser.url('https://testpages.herokuapp.com/styled/iframes-test.html');

	const iframe = await browser.$('#thedynamichtml');
	await browser.switchToFrame(iframe);
	await browser.switchToFrame(null);

	await browser.deleteSession()
})().catch(async (e) => {
	console.error(e)
	await browser.deleteSession()
})

```

**Comment by straker** [comment]
The main issue we have is that we can't detect if there is top level browsing context or not before we call `switchToFrame`, so we don't know when passing `null` will work or not. 

Also, the error isn't thrown by `switchToFrame` because there is no parent, it errors because the code assumes `id` is a String and tries to access something it shouldn't. Would it be possible to have the error message reflect that expectation and the docs mentioned it?

**Comment by christian-bromann** [comment]
fixed in #6924
